config/middleware.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const cors = require("cors");
+const helmet = require("helmet");
```

Comment:
Good job remembering to bring in helmet.  Many people forget security when learning all of these new things.